### PR TITLE
Display multiple panels per section

### PR DESCRIPTION
**Pull request by csmarchbanks**
We were overwriting the section each time we came across a new metric within the section. Instead, append the new panel. Appending to a nil slice will naturally create a new slice so this is safe to do.

### CODE REVIEW

File: ai-training-api/app/model_metrics.go [hunk 1]
```diff
@@ -348,7 +348,7 @@ func transformMetricsData(results []Result) GetModelMetricsResponse {
 			panels = append(panels, newPanel)
 		}
 
-		response.Sections[sectionName] = panels
+		response.Sections[sectionName] = append(response.Sections[sectionName], panels...)
```
Comment (annanay25) [inline ai-training-api/app/model_metrics.go:351]: It's a little confusing to have the metric namespace as part of the name, maybe we can split that out into a separate field? But also that can be handled in another issue

Comment (annanay25) [inline ai-training-api/app/model_metrics.go:351]: Can we add a test for this?

Comment (csmarchbanks) [reply ai-training-api/app/model_metrics.go:351]: Yeah, that is way too big of a change to want to do as part of this PR. That would probably change the entire log signature and probably request format too, not sure if that is something we want.

Comment (csmarchbanks) [reply ai-training-api/app/model_metrics.go:351]: Updated a test to catch this case as well!

**Review by annanay25 — APPROVED** [review]
LGTM!

**Pull request merged**